docs/book/src/user/quick-start.md

```diff
@@ -38,19 +38,44 @@ export KUBECONFIG=<...>
 
 **Minimum [kind] supported version**: v0.7.0
 
+**Starting from v0.8.0, you must export the variable `KIND_EXPERIMENTAL_DOCKER_NETWORK` to change the default network used by kind**
```

Comment:
Given that now there is a tab for v0.8, this sentence can be dropped

Reply:
done